x-pack/test_serverless/functional/test_suites/security/ftr/cloud_security_posture/agentless_api/create_agent.ts

```diff
@@ -41,6 +40,13 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     });
 
     it(`should create agentless-agent`, async () => {
+      if (process.env.TEST_CLOUD) {
+        // If the test is running in the Serverless Quality Gates
+        // expect that the Vault secrets were added to the environment
+        expect(process.env.CSPM_AWS_ACCOUNT_ID).to.be.ok();
+        expect(process.env.CSPM_AWS_SECRET_KEY).to.be.ok();
+      }
```

Comment:
This would require everyone running this serverless tests from their local against QA to have these env variables exported, which can be a hurdle. I guess what we need to trade off here is the inconvenience of having to manually export these env variables on the one side versus noise coming from dummy / non-existent AWS credentials in occasional local runs on the other side. What do you think?

Reply:
Yes,  this makes sense to remove it.  I was being a little too overly cautious to fail the SQG if the credentials were not found.   The alerts we have setup for Agentless in QA should raise the issue if the credentials fail to be passed.

I've removed the check-fail for the environment variables.